indexer/ai/src/test/java/com/google/cloudsearch/ai/TestBaseAISkill.java

```diff
@@ -0,0 +1,71 @@
+package com.google.cloudsearch.ai;
+
+import com.google.cloudsearch.exceptions.InvalidConfigException;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class TestBaseAISkill {
+
+    @Rule
+    public final ExpectedException exception = ExpectedException.none();
+
+    private JSONObject getTestSchema() {
+        JSONParser parser = new JSONParser();
+        JSONObject schema = null;
+        try {
+            schema = (JSONObject) parser.parse(new FileReader("./src/test/java/com/google/cloudsearch/ai/testFiles/BaseAISkill/sampleSchema.json"));
+        } catch (IOException e) {
```

Comment:
we should not catch exceptions in tests

Reply:
This function is not really a test, it is a helper function reading some sample schema